sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2494,11 +2494,14 @@ pi_result piQueueRelease(pi_queue Queue) {
         ZE_CALL(zeFenceDestroy, (MapEntry.second.ZeFence));
       }
       Queue->ZeCommandListFenceMap.clear();
-      ZE_CALL(zeCommandQueueDestroy, (Queue->ZeComputeCommandQueue));
-      Queue->ZeComputeCommandQueue = nullptr;
-      if (Queue->ZeCopyCommandQueue) {
-        ZE_CALL(zeCommandQueueDestroy, (Queue->ZeCopyCommandQueue));
-        Queue->ZeCopyCommandQueue = nullptr;
+
+      if (Queue->OwnZeCommandQueue) {
+        ZE_CALL(zeCommandQueueDestroy, (Queue->ZeComputeCommandQueue));
+        Queue->ZeComputeCommandQueue = nullptr;
```

Comment:
I think we should still null-ify these, even if we don't own the handles

Reply:
done.